chroma-manager/storage_server.repo

```diff
@@ -27,13 +27,13 @@ baseurl=https://downloads.hpdd.intel.com/public/e2fsprogs/latest/el7/
 enabled=1
 gpgcheck=0
 
-[managerforlustre-manager-for-lustre]
-name=Copr repo for manager-for-lustre owned by managerforlustre
-baseurl=https://copr-be.cloud.fedoraproject.org/results/managerforlustre/manager-for-lustre/epel-7-$basearch/
+[managerforlustre-manager-for-lustre-devel]
+name=Copr repo for manager-for-lustre-devel owned by managerforlustre
+baseurl=https://copr-be.cloud.fedoraproject.org/results/managerforlustre/manager-for-lustre-devel/epel-7-$basearch/
 type=rpm-md
 skip_if_unavailable=True
 gpgcheck=1
-gpgkey=https://copr-be.cloud.fedoraproject.org/results/managerforlustre/manager-for-lustre/pubkey.gpg
+gpgkey=https://copr-be.cloud.fedoraproject.org/results/managerforlustre/manager-for-lustre-devel/pubkey.gpg
```

Comment:
Not directly related to this patch, but probably should be generating all of this from [chroma-manager/tests/framework/utils/defaults.sh](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/467/commits/6c0cf3af708c44f8f3d6eb4e02f938f651d54e44#diff-06a3ab104091298eb2e45220cb3e6da4R113)